internal/controller/config/crd/bases/agent-install.openshift.io_agents.yaml

```diff
@@ -62,6 +62,8 @@ spec:
                 type: object
               hostname:
                 type: string
+              ignitionConfigOverride:
+                type: string
```

Comment:
could you please add a `description` to mention what this field is for (and the expected format). :pray: